design/fast_finality.md

```diff
@@ -66,17 +72,17 @@ the client behave as if the transactions were completely final).
 
 ### Production of rollups
 
-Every `x` blocks, the sequencer's host requests the creation of a rollup. The sequencer's enclave produces a rollup 
-containing all the light batches created since the last rollup, in a sparse Merkle tree structure. This rollup is 
-encrypted and signed, then sent to be included on the L1.
-
 A rollup is produced whenever one of the following conditions is met:
 
-* The number of transactions across all light-batches since the last rollup exceeds `y`
-* The value of all transactions across all light-batches since the last rollup exceeds `z`
-* The time since the last rollup was produced exceeds `w`
+* The number of transactions across all light-batches since the last rollup exceeds `x`
+* The value of all transactions across all light-batches since the last rollup exceeds `y`
+* The number of blocks since the last rollup was produced exceeds `z`
 
-`y`, `z` and `w` are configurable per network.
+`x`, `y` and `z` are configurable per network.
+
+To produce a rollup, the sequencer's host requests the creation of a rollup. The sequencer's enclave produces a rollup 
+containing all the light batches created since the last rollup, in a sparse Merkle tree structure. This rollup is 
```

Comment:
the sparse Mtree is an implementation possiblity. 
I'd say we better add the requirement for it. That bad behaviour ( like not including a signed LB), has to be provable.

Reply:
Added as a requirement